mpf/config_players/blinkenlight_player.py

```diff
@@ -0,0 +1,48 @@
+"""Light config player."""
+from copy import deepcopy
+from mpf.config_players.device_config_player import DeviceConfigPlayer
+from mpf.core.rgb_color import RGBColor
+from mpf.core.utility_functions import Util
+
+
+class BlinkenlightPlayer(DeviceConfigPlayer):
+
+    """Blinks lights different colors based on config."""
+
+    config_file_section = 'blinkenlight_player'
+    show_section = 'blinkenlights'
+    machine_collection_name = 'blinkenlights'
+    allow_placeholders_in_keys = True
+
+    def play(self, settings, context, calling_context, priority=0, **kwargs):
+        """Adds/removes colors to blink."""
+        del kwargs
+
+        for blinkenlight, s in settings.items():
+            action = s['action']
+            if action == 'add':
+                self._add_color(blinkenlight, s['color'], s['key'])
```

Comment:
We should pass priority here (`priority + s['priority']`).

Reply:
This makes me think. I believe that including priority here would include the mode's priority with the color.  This would mean that if a single blinkenlight had several colors added by different modes, each with a different priority, then the blinkenlight's colors might not all get shown, because priority might be overridden.  This seems strange to me.  The way I have it now is, the blinkenlight itself has a priority, and colors added to it all get the same priority regardless of the mode that added it.  That way, all of the blinkenlight's colors will either get shown or not shown as a group, instead of some colors being shown and others not shown due to the priority of the mode that added the color.

Another thing to think about is, what about the "off" color between cycles?  This isn't explicitly added by any mode, so what should its priority be?  If it's 0, then the "off" color might not get displayed at all which might look weird.

I'm not sure how this should be handled truthfully.

Reply:
Naming is a bit confusing here. `priority` on the light stack is one thing (which you already implemented). Another thing would be mode or stack priority or order. That would determine if red or blue comes first in your blinkenlight. This way the order should become deterministic (and based on mode priority). Makes sense?

Reply:
Got it. Thanks.